### PR TITLE
Docs update: Add EEG Filters chapter content

### PR DESCRIPTION
**Pull request by Anoushkaswain25**
Hello @lorforlinux, 
Description for EEG filters has been added.
1) Introduction on why filtering of EEG signal is important.
2) Block diagram of IIR filter is there along with its specification like passband and the response of the filter is attached.
3) Benefits of using IIR filter is there.

### CODE REVIEW

File: docs/eeg_filter.md [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
 
+**Block diagram of IIR filter**
 
-**Block diagram of IIR filter** 
+![Capture](https://user-images.githubusercontent.com/122666785/224307805-3c4e67ec-bc96-42e2-8335-df99cb492c3c.PNG)
```
Comment (lorforlinux) [inline docs/eeg_filter.md:8]: Please add the graphics files to the folder and reference from there.

File: docs/eeg_filter.md [hunk 2]
```diff
@@ -1,32 +1,24 @@
...
 
-<br><p align="center"> ![filter](https://user-images.githubusercontent.com/122666785/224324132-cd281ce0-e568-4528-b4c0-ef10c13de032.PNG)</p>
+**Response of the filter**
+
+![filter](https://user-images.githubusercontent.com/122666785/224324132-cd281ce0-e568-4528-b4c0-ef10c13de032.PNG)
 
-### <br>**Advantage of using IIR filter**</br>
+### Advantage of using IIR filter
 
 1) *Low implementation cost*: Requires less coefficients and memory than FIR filters in order to satisfy a similar set of specifications
```
Comment (lorforlinux) [inline docs/eeg_filter.md:21]: Did you validate this format of ordered list?

**Review by lorforlinux — COMMENTED** [review]
Some minor formatting updates are still required.

**Review by lorforlinux — APPROVED** [review]
@Anoushkaswain25 The image you have added is not at the right location. It should be included under `docs/` as we want to use it for docs. You have not updated the markdown to use the image you uploaded.

**Comment by Anoushkaswain25** [comment]
Hello sir,

Okay, I'll add it and see through it.

Regards
Anoushka

**Comment by lorforlinux** [comment]
We are shifting docs to a new site https://docs.upsidedownlabs.tech, closing this now.

**Pull request closed without merging**